features/facebookImport/rollup.config.js

```diff
@@ -26,6 +27,10 @@ export default (commandLineArgs) => {
             globals: externalPackages,
         },
         plugins: [
+            execute([
+                "cp node_modules/@polypoly-eu/podjs/dist/pod.js dist",
+                "node ../../platform/podjs/bin/genPodjs.js --podjs=./dist/pod.js --manifestJson=./src/static/manifest.json",
```

Comment:
The _genPodjs_ script should already know where `pod.js` is, in `../dist`. Couldn't this invocation just be:

```
node ../../platform/podjs/bin/genPodjs.js --manifestJson=./src/static/manifest.json
```

?

Or even better, you should be able to export the _genPodjs_ script as a binary from the _podjs_ package (adding a hash bang and removing the `.js` suffix while at it), so you could simply do:

```
genPodjs --manifestJson=./src/static/manifest.json
```

Reply:
in the case of `lexicon` it's not in /`dist` but in `/public` - thought it would be more universal to set them per feature to avoid issues

Reply:
Getting even crazier here, the _genPodjs_ script could even look for the manifest file itself in the current directory if it's not supplied - for now we could just check in `src/static/manifest.json` since we know it's always there. Then we'd have:

```
genPodjs > dist/pod.js
```

Reply:
Well, I did specify `dist` explicitly in all the example above, so the lexicon case should be covered by just changing it to `public` there.

My point is that _genPodjs_ knows its own location, and it can look into `../dist` relative to that to find the _source_ `pod.js` file. As for where it's supposed to put it, I agree the feature should specify that.